cobbler/modules/managers/isc.py

```diff
@@ -225,36 +215,215 @@ def write_dhcp_file(self):
             "yaboot": yaboot,
             "dhcp_tags": dhcp_tags
         }
-
         if self.logger is not None:
             self.logger.info("generating %s" % self.settings_file)
-        self.templar.render(template_data, metadata, self.settings_file)
+        self.templar.render(template_data, metadata, self.settings_file, None)
 
-    def regen_ethers(self):
+    def write_v6_config(self):
         """
-        ISC/BIND doesn't use this. It is there for compability reasons with other managers.
+        DHCP IPv6 files are written when ``manage_dhcp6`` is set in ``/etc/cobbler/settings``.
         """
-        pass
 
-    def sync_dhcp(self):
+        template_file = "/etc/cobbler/dhcp6.template"
+        blender_cache = {}
+
+        try:
+            f2 = open(template_file, "r")
+        except Exception:
```

Comment:
That is too lazy. Catch the exception you want to catch and handle them individually. Otherwise let it crash. Just silencing and re-rasing is not good style.

Reply:
I'll let this pass for now...